app/logic/serviceLearningCourses.py

```diff
@@ -43,7 +44,7 @@ def getSLProposalInfoForUser(user: User) -> Dict[int, Dict[str, Any]]:
 
         courseDict[course.id] = {"id":course.id,
                                  "creator":f"{course.createdBy.firstName} {course.createdBy.lastName}",
-                                 "name":course.courseName if course.courseName else course.courseAbbreviation,
+                                 "coursename": nameNumCombo(course.courseName, course.courseAbbreviation),                               
```

Comment:
Let's leave `name` and add `abbr` as keys, in addition to a `displayname` key. That way the template has more info to use.

Reply:
we have added both name, abbr and displayname as keys so that it can be used anywhere